src/main/java/com/google/netpcapanalysis/dao/UtilityPCAPDaoImpl.java

```diff
@@ -0,0 +1,94 @@
+package com.google.netpcapanalysis.dao;
+
+import com.google.netpcapanalysis.models.PCAPdata;
+import com.google.netpcapanalysis.interfaces.dao.UtilityPCAPDao;
+import com.google.netpcapanalysis.models.Flagged;
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.HashMap;
+import java.util.Collections;
+import java.util.Map;
+
+import java.math.BigInteger; 
+import java.security.MessageDigest; 
+import java.security.NoSuchAlgorithmException; 
+
+public class UtilityPCAPDaoImpl implements UtilityPCAPDao {
+
+  public UtilityPCAPDaoImpl() {
```

Comment:
make this a private constructor and have all methods as static, that is the typical design for a Utility class.